src/seedu/addressbook/AddressBook.java

```diff
@@ -199,25 +199,67 @@
      * ====================================================================
      */
     public static void main(String[] args) {
```

Comment:
okay, good! now think about what 'goodness' you lost by inlining methods. There are more things that make code 'nicer' besides methods.. appropriate comments, good variable names, coding style that enhances readability and debuggability.